GitUI/UserControls/RevisionGrid/Columns/CommitIdColumnProvider.cs

```diff
@@ -60,6 +60,15 @@ public override void OnCellPainting(DataGridViewCellPaintingEventArgs e, GitRevi
             }
         }
 
+        public override void OnCellFormatting(DataGridViewCellFormattingEventArgs e, GitRevision revision)
+        {
+            // Set the grid cell's accessibility text
+            if (!revision.IsArtificial)
```

Comment:
Why limit artificial?
Is this displayed somewhere other than for accessibility?
(I do not see that when commenting out the test.)

Reply:
> Why limit artificial?

We do not want to display "111...1" anywhere.

> Is this displayed somewhere other than for accessibility? (I do not see that when commenting out the test.)

No, it isn't. `OnCellPainting` draws the cell. I have just restored setting `e.Value`, which was removed by mistake in #8550.